devops/ansible/localhost

```diff
@@ -0,0 +1,2 @@
+[localhost]
+localhost
```

Comment:
Is this the inventory file?

Reply:
Yes, it was unnecessary and has been removed